library/src/test/java/com/datatorrent/lib/helper/OperatorContextTestHelper.java

```diff
@@ -44,22 +42,26 @@ protected DateFormat initialValue()
 
   };
 
-  public static class TestIdOperatorContext extends TestContext implements OperatorContext
+  public abstract static class TestIdOperatorContext extends TestContext implements OperatorContext
```

Comment:
Moving factory methods up would actually result in longer lines (OperatorContextTestHelper.contextOf() vs. TestIdOperatorContext.of()).